build.gradle

```diff
@@ -174,6 +174,7 @@ configure(javaCodeCheckedProjects) {
     /* Testing */
     tasks.withType(Test) {
         maxHeapSize = "3g"
+        jvmArgs = ['-XX:-UseCompressedOops']
```

Comment:
I'm surprised this helps.  It seems like this is the default as of Java 8:

    $ java -XX:+PrintFlagsFinal -version|grep UseCompressedOops
         bool UseCompressedOops                        := true                                {lp64_product}
         openjdk version "1.8.0_232"

Reply:
Yes this is to turn it off (its prefixed with a minus). Although the last rerun on evergreen just failed so it doesn't seem to help :(